packages/gestalt/src/Masonry/fullWidthLayout.js

```diff
@@ -52,13 +53,13 @@ const fullWidthLayout = <T>({
       } else {
         const col = mindex(heights);
         const top = heights[col];
-        const left = col * columnWidth + gutter / 2;
+        const left = col * columnWidthAndGutter + gutter / 2;
 
         heights[col] += height;
```

Comment:
just FYI - i checked and we do currently apply a slight gutter on mobile web so if we were to roll out the fullWidthMultiColumnLayout, it would cause an increased gap. it might be good to address that first (in a separate PR) and then land this one.